addon.xml

```diff
@@ -55,25 +55,30 @@
     <forum>https://forum.kodi.tv/showthread.php?tid=329767</forum>
     <source>https://github.com/CastagnaIT/plugin.video.netflix</source>
     <news>
+v0.16.4 (2020-01-28)
+-Up Next is now supported only from version 1.1.0 and up
+-Added czech language
+-Added "because you liked" to raccomendations menu
```

Comment:
Typo: recommendations

Reply:
haha thanks i have itanialized the word :)